src/stores/artists-store.js

```diff
@@ -0,0 +1,23 @@
+import { observable, action } from 'mobx'
+import axios from 'axios'
+
+class ArtistsStore {
+  @observable foundArtists = []
+
+  @action
+  getArtist(searchTerm) {axios
+    .get(`/api/artist/${searchTerm}`)
+    .then(response => {
+      artistsStore.foundArtists = response.data.artists.items.map(artist => ({
+        id: artist.id,
+        name: artist.name,
+        // url hardcoded for now, going to change later
+        img: artist.images['0'] ? artist.images['0'].url : 'http://089a12354f66f8c089ca-2b0da8e65ee035845e5cc0511bab26ba.r78.cf1.rackcdn.com/global/imagelib/distancelearning/no-img-6c91ea01d3af4559c726afc209ddec79d965cac9.png'
```

Comment:
Może dodaj jako stałą `defaultUrl` będzie czytelniej ;)

Reply:
> Jeszcze dwie pierdoły :P Na https://spotify-artist-finder.herokuapp.com/ jest akutalna wersja? Bo url wywala undefined, a nie powinno.
> 
> I jeszcze trochę kiepsko wygląda karta jak jest długa nazwa
> <img alt="screenshot 2018-11-14 at 18 06 35" width="1513" src="https://user-images.githubusercontent.com/28709559/48499381-9c20ad80-e838-11e8-8a77-f149bb5af5f9.png">
> 
> Na mobile nawet nie wygląda tragicznie, ale przydałoby się trochę dopracować :P I jako pomysł na optymalizację dodaj `debounce` do getArtist, tak żeby jak user wpisuje jakaś nazwę to nie robić requestów za każdym razem tylko jak skończy. Czyli np. jak w ciągu 0.25s nie wpisze kolejnej literki to dopiero wysłać request, ale to nie teraz :P

mam nawet pseudo metode  ala lodashowa z debounce juz napisana i chcialem to dodac w przyszlosci. Co do buga - to po zmianie na flexa, przy display:table tego nie bylo :/

Reply:
no tak, jasne, robilem na szybko bo wychodzilem dlatego tez dalem od razu sobie komentarz